rust/actyx/ax-aql/src/language/parser/mod.rs

```diff
@@ -664,12 +665,26 @@ impl FromStr for SimpleExpr {
     }
 }
 
-impl FromStr for Timestamp {
+pub struct AqlTimestamp(u64);
```

Comment:
slightly inconvenient, but this seems to be a hard limitation of the Rust trait system

Reply:
I found an alternative though, we can parse on the library side since this parsing method isn't actually used here.
This was the way I found to keep this working, but I found a library that does the parsing for us. I'll push a commit so you can see